yarn.lock

```diff
@@ -15877,6 +15877,13 @@ ovh-api-services@^11.0.1:
   dependencies:
     lodash "^4.17.20"
 
+ovh-api-services@^11.0.1:
```

Comment:
It's already bump on `develop` branch.

```sh
$ git rev-parse --short HEAD
db0eb5806ee0

$ yarn why ovh-api-services | grep Found
=> Found "ovh-api-services@11.0.1"
```

I'm rebasing this branch on `develop`.